frontend/src/components/modal/CreateBlogModal.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import './Modal.css';
+
+const CreateBlogModal = ({handleClose, show, children}) => {
+    const showHideClassName = show ? "card modal display-block" : "modal display-none";
+
+    return (
+        <div className={showHideClassName}>
```

Comment:
Lets update the layout of the form like bootstrap:
<img width="411" alt="Screen Shot 2019-09-20 at 6 04 23 PM" src="https://user-images.githubusercontent.com/5072991/65329215-763f0480-dbd1-11e9-82fb-e15a4447e6d9.png">

Reply:
done